Update sensor documentation for HomeWizard (#25604)
home-assistant/home-assistant.github.io: source/_integrations/homewizard.markdown

## Sensors

> **Sensors for the P1 meter, Energy socket and kWh meter**:

Reviewer: ```suggestion
Sensors for the P1 meter, Energy socket, and kWh meter:
```

> - **Total energy import/export (kWh)**: Total energy imported or exported since in installation. Each tariff has its own sensor (eg. T1, T2) and a sensor for the combined value.

Reviewer: ```suggestion
- **Total energy import/export (kWh)**: Total energy imported or exported since installation. Each tariff has its own sensor (e.g., T1, T2) and a sensor for the combined value.
```

> - **Active power (w)**: Active power that is measured on each phase.

Reviewer: ```suggestion
- **Active power (W)**: Active power that is measured on each phase.
```

> **Sensors for P1 meter, only available when smart meter exposes these values**:

Reviewer: ```suggestion
Sensors for P1 meter, only available when smart meter exposes these values:
```

> - **Gas usage (m3)**: Total gas used since the installation of the gas meter. A gas meter sends its measurement once every 5 minutes or per hour depending on the version of the smart meter.

Reviewer: ```suggestion
- **Gas usage (m³)**: Total gas used since the installation of the gas meter. A gas meter sends its measurement once every 5 minutes or per hour, depending on the version of the smart meter.
```

> - **Active tariff**: Current tariff that is used, can be used to keep consumption as low as possible during peak hours.

Reviewer: ```suggestion
- **Active tariff**: Current tariff that is used can be used to keep consumption as low as possible during peak hours.
```

Author: In my opinion this sentence is a bit weird now? What about:

```suggestion
- **Active tariff**: Current tariff that is used. Can be used to keep consumption as low as possible during peak hours.
```

> - **Active voltage (v)**: Active voltage that is measured on each phase.

Reviewer: ```suggestion
- **Active voltage (V)**: Active voltage that is measured on each phase.
```

> - **Active current (a)**: Active current that is measured on each phase.

Reviewer: ```suggestion
- **Active current (A)**: Active current that is measured on each phase.
```

> - **Active frequency (hz)**: Net frequency.

Reviewer: ```suggestion
- **Active frequency (Hz)**: Net frequency.
```

- **Voltage sags and swells**: Number of times a voltage sag or well has been detected.

> - **Power failures**: Two sensors that indicate the number of power failures that has been detected by the smart meter. One for all power failures and another for 'long' power failures.

Reviewer: ```suggestion
- **Power failures**: Two sensors that indicate the number of power failures that have been detected by the smart meter. One for all power failures and another for 'long' power failures.
```

- **Peak demand**: Belgium users are started to get charged for the peak usage per month (see [capaciteitstarief](https://www.fluvius.be/nl/thema/factuur-en-tarieven/capaciteitstarief)). Two sensors are available: One that shows the current quarterly average and another that shows the peak measured this month. Both these sensors are provided directly from the smart meter and can be used to keep the peak as low as possible.

> **Sensors for Water meter**:

Reviewer: ```suggestion
Sensors for Water meter:
```

> - **Active usage (l/min)**: Flow of water that is measured at that time.

Reviewer: ```suggestion
- **Active usage (L/min)**: Flow of water that is measured at that time.
```

> - **Total usage (m3)**: Total water usage since installation of the HomeWizard Water meter

Reviewer: ```suggestion
- **Total usage (m³)**: Total water usage since the installation of the HomeWizard Water meter.
```
